dev/sparktestsupport/modules.py

```diff
@@ -271,27 +271,27 @@ def __hash__(self):
     ],
 )
 
-connect = Module(
```

Comment:
Did you swap the order of `protobuf` and `connect`? The diff looks confusing :).

Reply:
yep, I swapped the order to let `protobuf` be able to use in `dependencies=[hive, avro, protobuf]`